### PR TITLE
fix: serverless app more method

### PR DESCRIPTION
**Pull request by echosoar**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: packages-serverless/serverless-app/src/utils.ts [hunk 1]
```diff
@@ -65,21 +65,76 @@ export const analysisDecorator = async (cwd: string) => {
...
+
+      if (!allFunc[funcName].handler) {
+        allFunc[funcName].handler = handler;
+      }
+
+      const trigger = func.functionTriggerName;
+      let isAddToTrigger = false;
+      const { path, method } = func.functionTriggerMetadata;
+      let methodList = [].concat(method || []);
+      if (methodList.includes('any') || methodList.includes('all')) {
```
Comment (czy88840616) [inline packages-serverless/serverless-app/src/utils.ts:100]: 这里应该拿不到 any 和 all 了。

File: packages-serverless/serverless-app/src/utils.ts [hunk 2]
```diff
@@ -65,21 +65,76 @@ export const analysisDecorator = async (cwd: string) => {
...
+        func.functionTriggerMetadata.method = 'any';
+        methodList = ['any'];
+      } else {
+        func.functionTriggerMetadata.method = methodList;
+      }
+      // 避免重复路径创建多个trigger
+      const httpTrigger = allFunc[funcName].events.find(event => {
```
Comment (czy88840616) [inline packages-serverless/serverless-app/src/utils.ts:107]: 现在重复路径会抛错，不可能出现。

**Review by czy88840616 — APPROVED** [review]

**Review by czy88840616 — CHANGES_REQUESTED** [review]
改改?

File: packages-serverless/serverless-app/src/utils.ts (hunk 2 above)
Comment (echosoar) [reply packages-serverless/serverless-app/src/utils.ts:107]: 同一个方法上面没法添加多个装饰器？@Get 和 @Post 都加在一个方法上面

**Pull request merged**